p2p/nodechecker_test.go

```diff
@@ -109,7 +109,7 @@ func TestHost_LocalAddressChange(t *testing.T) {
 
 			require.Eventually(t, func() bool {
 				return len(nodeB.host.Peerstore().Addrs(nodeA.host.ID())) == 1
-			}, 2*time.Second, 100*time.Millisecond, "nodeB should have 1 address of nodeA")
+			}, 4*time.Second, 100*time.Millisecond, "nodeB should have 1 address of nodeA")
```

Comment:
doesn't it seem strange that 2s is not enough for address to update?

Reply:
usually enough 100ms. But when i run `go test --count 100` one time get fail here. Probably, as we run multiply times, it generate high load at machine, and not enough time for exchange messages between nodes.

as we run it in github ci, and do not know in which condition instance - better set more time, in most cases it will finish early than in 4 sec. But for cases when performance is low - it may help.

Reply:
> Probably, as we run multiply times, it generate high load at machine, and not enough time for exchange messages between nodes.

isn't this unittest? what do you mean between nodes? 
without looking at the code, is there potentially lock contention?

Reply:
yes, this is unit test. Usually was used `mocknet` tool for test behavior in unit tests. But there too much mocks and you can't inject some modules.

So it actually this test create 2 nodes on same host, but on different ports. And after some actions with nodeA check that nodeB catch changes